client/packages/programs/src/JsonForms/common/components/Autocomplete.tsx

```diff
@@ -50,38 +76,33 @@ const UIComponent = (props: ControlProps) => {
     if (schemaOptions?.freeText) {
       // set the local data so that we have it in onClose; don't set it otherwise because it would
       // affect the normal behaviour, i.e. the input wouldn't clear if the value is not valid
-      setLocalData(value);
+      setLocalData({ value, label: value });
     }
   };
 
   const onChange = (
     _event: React.SyntheticEvent,
-    value: DisplayOption | null | string
+    option: DisplayOption | null
   ) => {
     let s = '';
-    if (typeof value === 'string') {
+    if (typeof option === 'string') {
```

Comment:
This doesn't trigger at all...

Reply:
that was part of why free text import was not working...